src/app/pages/network/network.component.ts

```diff
@@ -543,7 +545,7 @@ export class NetworkComponent implements OnInit, OnDestroy {
       name: 'manage',
       matTooltip: this.translate.instant('Manage'),
       onClick: (row: IpmiRow) => {
-        window.open(`http://${row.ipaddress}`);
+        this.redirect.open(`http://${row.ipaddress}`);
```

Comment:
The problem only happens when user is trying to access paths under truenas host. This is probably going to be a different ip, so warning is not needed here.
Same for ipmi form above.